etc/kayobe/kolla/config/prometheus/prometheus-alertmanager.yml

```diff
@@ -0,0 +1,31 @@
+global:
+  resolve_timeout: 5m
+  smtp_require_tls: true
+  slack_api_url: '{{ secrets_slack_notification_channel_url }}'
+route:
+  receiver: 'default'
+  group_by: [alertname, app]
+  group_interval: 10m
+  repeat_interval: 24h
+  routes:
+    - match:
+        severity: page
+      # Also generate Jira ticket
+      continue: true
+      receiver: page
+    - match:
+        severity: page
+      receiver: default
+receivers:
+  - name: 'default'
+    webhook_configs:
+    - url: 'http://localhost:9097/alert'
+      # Jiralert doesn't handle resolving issues, and complains if this is true
+      send_resolved: false
+  - name: 'page'
+    slack_configs: {% raw %}
+    - channel: '#iris-notifications'
+      title: '{{ '{{' }} .GroupLabels.alertname {{ '}}' }} alert has triggered'
+      text: '{{ '{{' }} range .Alerts {{ '}}' }}{{ '{{' }} .Annotations.description {{ '}}' }}\n{{ '{{' }} end {{ '}}' }} '
```

Comment:
I'm also surprised this works. Don't you need double quotes for the outer quotation?

`"{{ '{{' }} range .Alerts {{ '}}' }}{{ '{{' }} .Annotations.description {{ '}}' }}\n{{ '{{' }} end {{ '}}' }}"`

otherwise the first quote gets closed to form:

`'{{ '`

Reply:
This is how it is rendered when it is copied out to the monitoring node:
```
global:
  resolve_timeout: 5m
  smtp_require_tls: true
  slack_api_url: 'SOME_SECRET_URL'
route:
  receiver: 'default'
  group_by: [alertname, app]
  group_interval: 10m
  repeat_interval: 24h
  routes:
    - match:
        severity: page
      # Also generate Jira ticket
      continue: true
      receiver: page
    - match:
        severity: page
      receiver: default
receivers:
  - name: 'default'
    webhook_configs:
    - url: 'http://localhost:9097/alert'
      # Jiralert doesn't handle resolving issues, and complains if this is true
      send_resolved: false
  - name: 'page'
    slack_configs:
    - channel: '#iris-notifications'
      title: '{{ .GroupLabels.alertname }} alert has triggered'
      text: '{{ range .Alerts }}{{ .Annotations.description }}\n{{ end }} '
templates: []
```

Reply:
Longer term we should probably get rid of one of the templating steps.

Reply:
```
    slack_configs: {{ '{% raw %}' }}{% raw %}
    - channel: '#iris-notifications'
      title: '{{ .GroupLabels.alertname }} alert has triggered'
      text: '{{ range .Alerts }}{{ .Annotations.description }}\n{{ end }}'
{% endraw %}{{ '{% endraw %}' }}
``` is slightly nicer

Reply:
Thanks. I would have switched from the `template` to the `copy` module for this conf file in kolla-ansible, but it would break other users as it already existed before this.